docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md

```diff
@@ -15,40 +15,51 @@ The Stripe integration in Spryker is a part of the App Composition Platform and
 ## Supported business models
 The Stripe App supports the business-to-business (B2B) and business-to-consumer (B2C) models.
 
-## Prerequisites for using the Stripe app
+## Features
+The Stripe App currently has the following features:
+1. Interface within Spryker ACP Catalog to connect with Stripe: Users can easily connect to Stripe from the App Composition Platform.
+2. Easy switch between Test & Live mode: Users have the option to test payments in either mode.
+3. Responsive Redirect Payment Page: Once a connection is set up between Spryker & Stripe, End users checking out are redirected to a Spryker-hosted payment page where they can view activated payment methods. This works both on web and mobile.
+4. View Payment Methods activated in your Stripe Dashboard
```

Comment:
don't quite get what this means. What is the "View payment methods"?

Reply:
It means that payment emthods activated within Stripe can be viewed on the redirect payment page